bedrock/mozorg/views.py

```diff
@@ -254,11 +254,10 @@ def home_tweets(locale):
 def home(request, template='mozorg/home/home.html'):
     locale = l10n_utils.get_locale(request)
 
-    # en-US gets new template
-    # not using locale-specific templates naming scheme yet as page/test results
-    # are still in flux
-    if locale == 'en-US':
-        template = 'mozorg/home/home-en-US.html'
+    if lang_file_is_active('mozorg/home/home-2016', l10n_utils.get_locale(request)):
```

Comment:
This works for me, but my understanding from the discussion with Jen (and the bug) was that we wanted to drop the old homepage and redirect to en-US for incomplete locale (on Oct 28).